### PR TITLE
Add sidekiq failures gem

### PR DESCRIPTION
**Pull request by cveneziani**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,52 +2,53 @@ source 'https://rubygems.org'
...
+gem 'sidekiq-failures', '~> 0.4.5'
+gem 'simple_form',      '~> 3.2.0'
+gem 'sinatra',          '~> 1.4.6', require: nil
+gem 'slim',             '~> 3.0.6'
+gem 'valid_email',      '~> 0.0.11'
+gem 'validate_url',     '~> 1.0.2'
 
 # Assets gems
 source 'https://rails-assets.org' do
+  gem 'rails-assets-clipboard',   '1.5.3'
```
Comment (kdisneur) [inline Gemfile]: :100:

**Comment by kdisneur** [comment]
:+1:

File: Gemfile (hunk 1 above)
Comment (cveneziani) [inline Gemfile]: :grimacing:

**Comment by alaibe** [comment]
:+1:

**Pull request merged**